dkg/pedersen/handler.go

```diff
@@ -513,5 +656,321 @@ func (h *Handler) handleDeal(ctx context.Context, msg types.Deal, out mino.Sende
 		}
 	}
 
+	dela.Logger.Debug().Msgf("%s is certified", h.me)
+
 	return nil
 }
+
+func (h *Handler) announceDkgPublicKey(isCommonNode bool, out mino.Sender, from mino.Address) error {
```

Comment:
it's a bit weird from an API perspective that `isCommonNode` gets passed in, but `isNewNode` gets computed internally.